<?xml version="1.0" encoding="UTF-8"?>
<ui version="4.0">
 <class>Form</class>
 <widget class="QWidget" name="Form">
  <property name="geometry">
   <rect>
    <x>0</x>
    <y>0</y>
    <width>741</width>
    <height>643</height>
   </rect>
  </property>
  <property name="windowTitle">
   <string>Form</string>
  </property>
  <layout class="QVBoxLayout" name="verticalLayout">
   <item>
    <layout class="QHBoxLayout" name="horizontalLayout_2">
     <item>
      <widget class="GraphicsLayoutWidget" name="graphicsViewStreamingData">
       <property name="sizePolicy">
        <sizepolicy hsizetype="MinimumExpanding" vsizetype="MinimumExpanding">
         <horstretch>0</horstretch>
         <verstretch>0</verstretch>
        </sizepolicy>
       </property>
      </widget>
     </item>
     <item>
      <widget class="HistogramLUTWidget" name="histogramAdjust" native="true"/>
     </item>
    </layout>
   </item>
   <item>
    <layout class="QHBoxLayout" name="horizontalLayoutConnect">
     <item>
      <widget class="QLineEdit" name="textBoxPublisherAddress"/>
     </item>
     <item>
      <widget class="QPushButton" name="buttonConnectPublisher">
       <property name="text">
        <string>Connect</string>
       </property>
       <property name="default">
        <bool>false</bool>
       </property>
      </widget>
     </item>
    </layout>
   </item>
   <item>
    <layout class="QHBoxLayout" name="horizontalLayoutStatus">
     <item>
      <widget class="QLabel" name="labelStatus">
       <property name="sizePolicy">
        <sizepolicy hsizetype="Fixed" vsizetype="Preferred">
         <horstretch>0</horstretch>
         <verstretch>0</verstretch>
        </sizepolicy>
       </property>
       <property name="text">
        <string>Status:</string>
       </property>
       <property name="wordWrap">
        <bool>true</bool>
       </property>
      </widget>
     </item>
     <item>
      <widget class="QLabel" name="labelStatusInfo">
       <property name="text">
        <string/>
       </property>
      </widget>
     </item>
    </layout>
   </item>
  </layout>
 </widget>
 <customwidgets>
  <customwidget>
   <class>GraphicsLayoutWidget</class>
   <extends>QGraphicsView</extends>
   <header>pyqtgraph.h</header>
  </customwidget>
  <customwidget>
   <class>HistogramLUTWidget</class>
   <extends>QWidget</extends>
   <header>pyqtgraph.h</header>
   <container>1</container>
  </customwidget>
 </customwidgets>
 <tabstops>
  <tabstop>textBoxPublisherAddress</tabstop>
  <tabstop>buttonConnectPublisher</tabstop>
  <tabstop>graphicsViewStreamingData</tabstop>
 </tabstops>
 <resources/>
 <connections>
  <connection>
   <sender>textBoxPublisherAddress</sender>
   <signal>returnPressed()</signal>
   <receiver>buttonConnectPublisher</receiver>
   <slot>click()</slot>
   <hints>
    <hint type="sourcelabel">
     <x>281</x>
     <y>260</y>
    </hint>
    <hint type="destinationlabel">
     <x>326</x>
     <y>260</y>
    </hint>
   </hints>
  </connection>
 </connections>
</ui>
